lib/tasks/export_portage.rake

```diff
@@ -69,6 +73,9 @@ namespace :export_production_data do
             end
             QuestionFormat.all.each do |question_formats| 
                 excluded_keys = ['created_at','updated_at'] 
+                if question_formats.id == 7
+                    question_formats.option_based = FALSE
```

Comment:
Is this still needed? You are already setting the `option_based` value to `false` on line 106 from db/seeds/seeds_1.rb

Reply:
@pengyin-shan

Reply:
@lagoan I left it here because the current production database is still set the `option_based` to TRUE, so in case I need to re-generate the seed files (including seeds_1.rb) again (for example, re-run this task before the sandbox deployment), the value in seeds_1b.rb will be overwritten. I could remove it but need to put a note for this part.

Reply:
I put a note at #167 for asking Neil to modify the current production database, but I was thinking to do it after the sandbox deployment...probably I can do it now then remove this line?

Reply:
@pengyin-shan sounds good!